app/allocations/middleware.js

```diff
@@ -49,14 +50,38 @@ function setPagination(req, res, next) {
   next()
 }
 async function setAllocationsSummary(req, res, next) {
-  const { dateRange } = req.params
-  const value = await allocationService.getCount({ dateRange })
+  const { dateRange } = res.locals
+  const allocationsCount = await allocationService.getCount({ dateRange })
```

Comment:
We can remove this commit too as it's replicating what the main dashboard now does. I'm changing allocations to use the shared logic around listing results and enabling them to see the shared dashbard.

That will avoid us adding more to this complicated area for now. So I'd say pull this commit into a separate branch too for now and focus this change on just supporting cancelling an allocation.